crates/router/Cargo.toml

```diff
@@ -84,6 +84,8 @@ url = { version = "2.4.0", features = ["serde"] }
 utoipa = { version = "3.3.0", features = ["preserve_order", "time"] }
 utoipa-swagger-ui = { version = "3.1.3", features = ["actix-web"] }
 uuid = { version = "1.3.3", features = ["serde", "v4"] }
+openssl = "0.10.55"
```

Comment:
Can we avoid `openssl` as a direct dependency if possible? Can we use `rustls` instead, or any other crate if it addresses your use case?

Reply:
Sure I will explore rustle and other crates to do the necessary changes. Can I take this changes is in the next pr as this pr is of priority ?